spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    # 昨日に完了したタスク
+    trait :task_done_yesterday do
+      status { 'done' }
```

Comment:
Symbolで記載して、String型のカラムでないことを強調しましょう。
```suggestion
      status { :done }
```

Reply:
ce4aeba こちらSymbolに修正しました。

Reply:
ce4aeba こちらSymbolに修正しました。

Reply:
ce4aeba こちらSymbolに修正しました。他の方が見て分かりやすいようにしているんですね。